### DISCUSSION TITLE
Blinking

### DISCUSSION THREAD

**Discussion by Bowarc** [lptstr/winfetch#118] [Q&A] [answered]
I didn't used powershell a lot so I sort of understand the code of this app, pretty cool tbh, but i have a question.

How do the -blink option work ? how is that possible that the logo still blink after the code stopped its exectution ?

I rly don't understand how is that possible, i looked up a bit for blinking text in powershell, but diddn't found any of the examples in your app

**Comment by jcwillox** [comment, marked as answer]
The blinking effect is an ANSI sequence used for styling text in terminals. So, just like bold text would remain bold after the program runs so does the blink effect. The terminal handles the blinking the program just tells it how to display the text. The wiki page has tonnes more info on [ANSI sequences](https://en.wikipedia.org/wiki/ANSI_escape_code).

**Reply by Bowarc** [reply]
Ooh i see, thanks for the fast and clear answer !
